src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -930,16 +930,17 @@ void GcsActorManager::LoadInitialData(const EmptyCallback &done) {
           RAY_CHECK(unresolved_actors_[owner_node][owner_worker]
                         .emplace(actor->GetActorID())
                         .second);
-          if (!actor->IsDetached() && worker_client_factory_) {
-            // This actor is owned. Send a long polling request to the actor's
-            // owner to determine when the actor should be removed.
-            PollOwnerForActorOutOfScope(actor);
-          }
         } else if (item.second.state() == ray::rpc::ActorTableData::ALIVE) {
           created_actors_[actor->GetNodeID()].emplace(actor->GetWorkerID(),
                                                       actor->GetActorID());
         }
 
+        if (!actor->IsDetached()) {
```

Comment:
Can you explain this part a little more? The new behavior is completely different from the previous one, and I don't understand how this fixes the issue.

Reply:
When GCS restarts, `PollOwnerForActorOutOfScope` is not called for the actor of alive, so the owner's actor will not go to `DestroyActor` even if the owner is dead.